js/anybits.js

```diff
@@ -7,35 +7,35 @@ const bitsane = require ('./bitsane.js');
 module.exports = class anybits extends bitsane {
     describe () {
         return this.deepExtend (super.describe (), {
-            'id': 'anybits',
-            'name': 'Anybits',
-            'countries': [ 'IE' ], // Ireland
-            'has': {
-                'fetchCurrencies': true,
-                'fetchTickers': true,
-                'fetchOpenOrders': true,
-                'fetchDepositAddress': true,
-                'withdraw': true,
+            id: 'anybits',
+            name: 'Anybits',
+            countries: ['IE'], // Ireland
+            has: {
+                fetchCurrencies: true,
+                fetchTickers: true,
+                fetchOpenOrders: true,
+                fetchDepositAddress: true,
+                withdraw: true,
             },
-            'urls': {
-                'logo': 'https://user-images.githubusercontent.com/1294454/41388454-ae227544-6f94-11e8-82a4-127d51d34903.jpg',
-                'api': 'https://anybits.com/api',
-                'www': 'https://anybits.com',
-                'doc': 'https://anybits.com/help/api',
-                'fees': 'https://anybits.com/help/fees',
+            urls: {
+                logo: 'https://user-images.githubusercontent.com/1294454/41388454-ae227544-6f94-11e8-82a4-127d51d34903.jpg',
+                api: 'https://anybits.com/api',
+                www: 'https://anybits.com',
+                doc: 'https://anybits.com/help/api',
+                fees: 'https://anybits.com/help/fees',
             },
-            'api': {
-                'public': {
-                    'get': [
+            api: {
+                public: {
+                    get: [
                         'assets/currencies',
                         'assets/pairs',
                         'ticker',
                         'orderbook',
                         'trades',
-                    ],
+                   ],
```

Comment:
Removing the space before this bracket will destroy Python padding, unfortunately...